### ISSUE TITLE
installedPackages fail to deploy

### ISSUE THREAD

**Issue by mattsones** [kevinohara80/grunt-ant-sfdc#32]
I get this error when attempting to deploy an installed package:
> Fatal error: installedPackages is not a valid metadata type

config:

```
'antdeploy': {
      options: {
        apiVersion: '36.0'
      },
      installedPackages: {
        options: {
          root: 'deploy/',
          user: ***********,
          pass: ***********,
          token: ***********
        },
        pkg: {
          installedPackages: ['*']
        }
      }
```

Here's the full console output:
```
c:\work\salesforce_ant\OptimizeDev>grunt antdeploy:installedPackages --buildtarget=sbsand
Running "antdeploy:installedPackages" (antdeploy) task
Deploy Target -> installedPackages
User -> *************
**Fatal error: installedPackages is not a valid metadata type**
```
deploy folder structure:
![deploy folder structure](https://cloud.githubusercontent.com/assets/7633346/15195898/ea4742d6-1797-11e6-9ffb-9813c6be6ed0.PNG)

package.xml:
```
<?xml version="1.0" encoding="UTF-8"?>
<Package xmlns="http://soap.sforce.com/2006/04/metadata">
    <types>
        <members>skuid</members>
        <name>InstalledPackage</name>
    </types>
    <version>36.0</version>
</Package>
```
skuid.installedPackage:
```
<?xml version="1.0" encoding="UTF-8"?>
<InstalledPackage xmlns="http://soap.sforce.com/2006/04/metadata">
    <versionNumber>8.6</versionNumber>
</InstalledPackage>
```

**Comment by thegogz** [comment]
It doesn't look like that's a supported metadata type for the lib 

You can see the supported types here https://github.com/kevinohara80/grunt-ant-sfdc/blob/master/lib/metadata.json

**Comment by mattsones** [comment]
How difficult is it to add support for a metadata type? Is it as simple as adding to the list? The Force.com Migration Tool seems to handle Installed Packages just fine.

**Comment by mattsones** [comment]
I added a pull request for this: https://github.com/kevinohara80/grunt-ant-sfdc/pull/33

**Comment by thegogz** [comment]
That should allow it to be used alright as the salesforce-ant.jar included is version 29 which supports the type however I don't have write access to this repo to merge your pull request you could always fork and use your forked version until that's merged

**Comment by kevinohara80** [comment]
I'll get this merged today. Thanks for the PR

**Comment by mattsones** [comment]
Awesome. Thanks!

**Comment by kevinohara80** [comment]
Merged and published in 1.3.0!
